src/main/java/org/ow2/proactive/catalog/report/CatalogObjectReportPDFGenerator.java

```diff
@@ -80,62 +100,150 @@
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Project Name");
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Object Name");
+        createDataHeaderCell(factHeaderrow, (100 / 8f) * 2, "Description");
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Kind");
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Content Type");
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Icon");
+        table.addHeaderRow(factHeaderrow);
+    }
 
-        String currentBucketName = "";
+    private void createDataHeaderCell(Row<PDPage> factHeaderrow, float width, String title) {
+        Cell<PDPage> cell;
+        cell = factHeaderrow.createCell(width, title);
+        cell.setFontSize(6);
+        cell.setFillColor(java.awt.Color.decode("#F3F3F4"));
+        cell.setAlign(HorizontalAlignment.CENTER);
+    }
 
+    private List<String[]> getCatalogObjects(SortedSet<CatalogObjectMetadata> orderedObjectsPerBucket)
+            throws IOException {
+        //Create the data
+        List<String[]> data = new ArrayList<String[]>();
         for (CatalogObjectMetadata catalogObject : orderedObjectsPerBucket) {
 
-            currentBucketName = separateBucketsWithEmptyRow(data, currentBucketName, catalogObject);
+            data.add(new String[] { catalogObject.getBucketName(), getProjectName(catalogObject),
+                                    catalogObject.getName(), getDescription(catalogObject), catalogObject.getKind(),
+                                    catalogObject.getContentType(), getIcon(catalogObject) });
 
-            data.add(new ArrayList<>(Arrays.asList(catalogObject.getBucketName(),
-                                                   catalogObject.getProjectName(),
-                                                   catalogObject.getName(),
-                                                   catalogObject.getKind(),
-                                                   catalogObject.getContentType())));
         }
+
+        return data;
+
     }
 
-    private String separateBucketsWithEmptyRow(List<List> data, String currentBucketName,
-            CatalogObjectMetadata catalogObject) {
-        if (StringUtils.isEmpty(currentBucketName) || !currentBucketName.equals(catalogObject.getBucketName())) {
-            currentBucketName = catalogObject.getBucketName();
-            data.add(new ArrayList<>(EMPTY_ROW));
-        }
-        return currentBucketName;
+    private String getProjectName(CatalogObjectMetadata catalogObject) {
+        return Optional.ofNullable(catalogObject.getProjectName()).orElse("");
+    }
+
+    private String getDescription(CatalogObjectMetadata catalogObject) {
+        return catalogObject.getMetadataList()
+                            .stream()
+                            .filter(metadata -> metadata.getKey().equals("description"))
+                            .map(metadata -> metadata.getValue())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)